boefjes/boefjes/config.py

```diff
@@ -109,6 +111,19 @@ class Settings(BaseSettings):
         ..., examples=["secret"], description="Bytes JWT login password", validation_alias="BYTES_PASSWORD"
     )
 
+    encryption_middleware: EncryptionMiddleware = Field(
+        EncryptionMiddleware.IDENTITY,
+        description="Toggle used to configure the encryption strategy",
+        examples=["IDENTITY", "NACL_SEALBOX"],
+    )
+
+    katalogus_private_key_b64: str = Field(
+        "", description="Base64 encoded private key used for asymmetric encryption of settings"
+    )
+    katalogus_public_key_b64: str = Field(
+        "", description="Base64 encoded public key used for asymmetric encryption of settings"
+    )
```

Comment:
I know we have the same thing in `bytes`, but imho this would be better:
- Remove the `_b64` prefix; it doesn't add much since the documentation is already clear on this
- Use`Base64Str` type instead of `str`; this gives free validation and documentation
```suggestion
    katalogus_private_key: Base64Str = Field(
        "", description="Base64 encoded private key used for asymmetric encryption of settings"
    )
    katalogus_public_key: Base64Str = Field(
        "", description="Base64 encoded public key used for asymmetric encryption of settings"
    )
```

Reply:
Cool wrt the type, not sure if changing this environment variable name is worth the migration risk involved when users are upgrading their installation and do not read the documentation well enough..

Reply:
Ah right that's `True`

Reply:
Done

Reply:
> Just checked, but it appears the Base64str and Base64Bytes objects are not backward compatible with the values..

That's fine with me. I was just wondering in what sense it's not backwards compatible with (older) values? They're just strings which will be validated on instantiation